website/src/components/Tippy.js

```diff
@@ -1,12 +1,139 @@
-import Tippy from '@tippy.js/react'
-import 'tippy.js/dist/tippy.css'
-import 'tippy.js/dist/themes/light.css'
-import 'tippy.js/dist/themes/light-border.css'
-import 'tippy.js/dist/themes/google.css'
+import tippy from '../../../src/js'
+import '../../../index.css'
+import '../../../themes/light.css'
+import '../../../themes/light-border.css'
+import '../../../themes/google.css'
+import React from 'react'
+import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+
+// These props are not native to `tippy.js` and are specific to React only.
+const REACT_ONLY_PROPS = ['children', 'onCreate', 'isVisible', 'isEnabled']
+
+// Avoid Babel's large '_objectWithoutProperties' helper function.
+function getNativeTippyProps(props) {
+  return Object.keys(props).reduce((acc, key) => {
+    if (REACT_ONLY_PROPS.indexOf(key) === -1) {
+      acc[key] = props[key]
+    }
+    return acc
+  }, {})
+}
+
+class Tippy extends React.Component {
```

Comment:
I assume this is temporarily here until `@tippy.js/react@2` is out, right?

Reply:
Yup